### PR TITLE
Fix broken documentation links

### PR DESCRIPTION
**Pull request by anjbur**

### CODE REVIEW

File: src/Simulation/Native/CMakeLists.txt [hunk 1]
```diff
@@ -44,6 +44,9 @@ else (MSVC)
   set(CMAKE_CXX_FLAGS "-static-libgcc")
 endif (MSVC)
 
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /Qspectre")
```
Comment (tcNickolas) [inline src/Simulation/Native/CMakeLists.txt:47]: This seems unrelated to the links modification? If it's a necessary change, I'd recommend doing it separately, figuring out why the build broke after only fixing some links would be a pain :-)

File: src/Simulation/Simulators/QuantumSimulator/Qubit.cs [hunk 2]
```diff
@@ -20,10 +20,16 @@ public QSimQubit(int id, uint simulatorId) : base(id)
                 this.SimulatorId = simulatorId;
             }
 
+            public QSimQubit(int id, uint simulatorId, string label) : this(id, simulatorId) {
```
Comment (tcNickolas) [inline src/Simulation/Simulators/QuantumSimulator/Qubit.cs:23]: This one also looks like it should be a separate change

**Review by tcNickolas — CHANGES_REQUESTED** [review]
Two files have changes unrelated to fixing links - if they're needed, I recommend excluding them from this PR and doing them separately.

File: src/Simulation/Native/CMakeLists.txt (hunk 1 above)
Comment (anjbur) [reply src/Simulation/Native/CMakeLists.txt]: Reverted, that was my mistake. Picked up some changes from other work that shouldn't have been included here.

File: src/Simulation/Simulators/QuantumSimulator/Qubit.cs (hunk 2 above)
Comment (anjbur) [reply src/Simulation/Simulators/QuantumSimulator/Qubit.cs]: Reverted as well, same mistake

**Review by tcNickolas — APPROVED** [review]
